package/package.json

```diff
@@ -29,13 +29,46 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.0.0"
+		"@vitejs/plugin-react": "^4.2.1",
+		"astro": "^4.4.1",
+		"preact": "^10.19.4",
+		"react": "^18.2.0",
+		"solid-js": "^1.8.15",
+		"svelte": "^4.2.11",
+		"vue": "^3.4.19"
+	},
+	"peerDependenciesMeta": {
+		"@vitejs/plugin-react": {
+			"optional": true
+		},
+		"preact": {
+			"optional": true
+		},
+		"react": {
+			"optional": true
+		},
+		"solid-js": {
+			"optional": true
+		},
+		"svelte": {
+			"optional": true
+		},
+		"vue": {
+			"optional": true
+		}
 	},
 	"devDependencies": {
 		"@types/node": "^20.11.0",
+		"@vitejs/plugin-react": "^4.2.1",
```

Comment:
why are the new deps in both optional and dev?

Reply:
When you `pnpm add --save-peer` a package it automatically puts it in dev deps too. I guess because you'll want it during development, just don't care in prod?

Reply:
It shouldn't be needed. That is for npm version 6 and below, pnpm should work fine with just the peer deps.